code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -994,6 +1008,44 @@ $(function() {
 		setCookie('highlightcolor', opts.highlightColor, 365);
 	});
 
+	$('#messageLimit').click(function(e) {
+		if ($('.popup .messageLimit').is(':visible')) {return;}
+		var popupContent = '<div class="head">Chat Message Limit</div>' +
+			'<div class="messageLimitPopup" id="messageLimitPopup">' +
+				'<div>Choose the limit of messages in the chat. Default value is '+opts.messageLimitMin+', min is '+opts.messageLimitMin+', max is '+opts.messageLimitMax+'. ' +
+					'If limit is reached, oldest messages at the top will be deleted. Higher limits may or may not be laggy during very long rounds.</div>' +
+				'<form id="messageLimitForm">' +
+				'<div><input type="text" name="messageLimitInput" id="messageLimitInput" class="messageLimitInput" maxlength="255" value="'+(opts.messageLimit ? opts.messageLimit : '')+'" /></div>' +
+					'<div><input type="submit" name="messageLimitSubmit" id="messageLimitSubmit" class="messageLimitSubmit" value="Save" /></div>' +
+				'</form>' +
+			'</div>';
+		createPopup(popupContent, 250);
+	});
+
+	$('body').on('submit', '#messageLimitForm', function(e) {
+		e.preventDefault();
+
+		var limit = $('#messageLimitInput').val();
+		limit = parseInt(limit);
+
+		if(isNaN(limit) || limit < opts.messageLimitMin) {
+			limit = opts.messageLimitMin
+			internalOutput('<span class="internal boldnshit">Message limit invalid or below min value.</span>', 'internal');
+		}
+		if(limit > opts.messageLimitMax) {
+			limit = opts.messageLimitMax
+			internalOutput('<span class="internal boldnshit">Message limit above max value.</span>', 'internal');
+		}
+
+		opts.messageLimit = limit
+		internalOutput('<span class="internal boldnshit">Message limit set to: '+opts.messageLimit+'</span>', 'internal');
```

Comment:
```suggestion
		internalOutput('<span class="internal boldnshit">Message limit set to '+opts.messageLimit+'</span>', 'internal');
```
i assume will output as "Message limit set to 1024."